DEVELOPMENT.md

```diff
@@ -462,10 +462,84 @@ This will compile binaries for all supported platforms, `deb`, `rpm` and Snapcra
 > Note: The operating system and architecture represent the name of the directory in `dist` in which the binaries are placed.
 > For example, the binaries for Darwin x64 (macOS) will be located at `dist/darwin_amd64`.
 
-Releasing a new version consists of the following steps:
+A new version can be released from the `master` branch or a `backport` branch. The necessary steps for each are detailed below.
 
-1. Creating a `release/<version>` branch off `master` or `backport/<minor>` (further, called "release branch") (e.g. `release/v3.2.1` or `backport/v3.2`).
-2. Updating the `CHANGELOG.md` file:
+> Note: To get the target version, you can run `version=$(./mage version:bumpXXX version:current)`, where xxx is the type of new release (minor/patch/RC).
```

Comment:
This is explained below; probably move that section up

Reply:
Added a reference to it instead.